modules/ding_availability/ding_availability.install

```diff
@@ -36,3 +17,12 @@ function ding_availability_update_7000() {
     ->condition('module', 'ding_availability')
     ->execute();
 }
+
+/**
+ * Unlock fields, see ding_entity_unlock_fields().
+ */
```

Comment:
Should the see have been a @see on it's one line for those who generate the doc-blocks!